### DISCUSSION TITLE
Overriding MPD_HOST and MPD_PORT?

### DISCUSSION THREAD

**Discussion by eriqjaffe** [jcorporation/myMPD#549] [Q&A] [answered]
I run two separate instances of MPD and myMPD.  With v7 I could set the host (each MPD instance uses a separate socket) and port in the appropriate .conf files.  How do I do that in v8?

**Comment by jcorporation** [comment, marked as answer]
You should start each instance with a different workdir and http/ssl Port and change the mpd instance in the webgui. At first startup the environment MPD_HOST and MPD_PORT is respected.

This should work, refer to the documentation for all options.
```
export MPD_HOST=/run/mpd1/socket
export MYMPD_HTTP_PORT=8000
export MYMPD_SSL=false
mympd -w /var/lib/mympd1
```

I will add this scenario to the documentation.

**Reply by eriqjaffe** [reply]
Thanks, that got it - Additionally I was able to get both instances running as services by copying /usr/lib/systemd/system/mympd.service to mympd1.service, adding the appropriate workdir option to ExecStart and then enabling/starting the mympd1 service.
